<?xml version="1.0" encoding="utf-8"?>

<launch>

  <!-- Load Parameter -->
  
  <param name="yolov5_path"       value="$(find yolov5_ros)/yolov5"/>
  <param name="use_cpu"           value="false" />

  <!-- Start yolov5 and ros wrapper -->
  <node pkg="yolov5_ros" type="pointer.py" name="pointer_ros" output="screen" >
    
    <param name="image_topic"       value="/camera/color/image_raw" />
    <param name="depth_topic"       value="/camera/depth/image_rect_raw" />
    <param name="pointcloud_topic"       value="/voxel_cloud" />
    <param name="position_topic"         value="/yolov5/BoundingBoxes" />
    <param name="camera_frame"       value="camera_color_frame"/>
  </node>

</launch>
